src/main/webapp/search-include-fragment.xhtml

```diff
@@ -545,6 +545,7 @@
                             <h:outputText value="#{bundle['dataset.versionUI.inReview']}" styleClass="label label-success" rendered="#{result.inReviewState}"/>
                             <h:outputText value="#{bundle['dataset.versionUI.unpublished']}" styleClass="label label-warning" rendered="#{result.unpublishedState}"/>
                             <h:outputText value="#{bundle['dataset.versionUI.deaccessioned']}" styleClass="label label-danger" rendered="#{result.deaccessionedState}"/>
+                            <h:outputText value="#{result.externalStatus}" styleClass="label label-info" rendered="#{!empty result.externalStatus and SearchIncludeFragment.canPublishDataset(result.entityId)}"/>
```

Comment:
I'm just putting this comment at the end (where I look for them)... some tests would be nice.

Reply:
I added some tests of setting the curation label set on a dataverse and then trying to set a label from that set on a dataset in the dataverse. Hopefully the tests will pass - awaiting the automated results.

Reply:
Fantastic! Thanks for adding tests!

Reply:
+1 for tests